Samples/excel-shared-runtime-scenario/src/functions/functions.js

```diff
@@ -49,3 +49,6 @@ function add(first, second) {
     throw new CustomFunctions.Error(CustomFunctions.ErrorCode.notAvailable);
   }
 }
+
+CustomFunctions.associate("ADD", add);
+CustomFunctions.associate("GETDATA", getData);
```

Comment:
I'm surprised to see that these weren't already using the associate() API. We recommend they use the API in our documentation, but I guess it wasn't required? Is that correct?

Reply:
I removed it from the onReady in taskpane.js and moved it here.